addon/utils/selection.js

```diff
@@ -63,24 +62,27 @@ export default {
    * @param {Function} itemComparator - comparator for items
    */
   /* eslint-disable complexity */
-  range (items, selectedItems, item, rangeState, itemComparator) {
+  range (items, selectedItems, item, rangeState, itemComparator, itemKey) {
     // If an anchor isn't set, then set the anchor and exit
     const rangeAnchor = rangeState['anchor']
-    if (isNone(rangeAnchor)) {
```

Comment:
isNone works when you set items as null.
my itemComparator was getting lhs.get('key') === rhs.get('key') problems due to you can't .get on null items.
I removed setting items as null and set them to empty ember objects. If a .get fails on that, at least you get undefined.